distributed/distributed.yaml

```diff
@@ -88,7 +93,7 @@ distributed:
   ###################
 
   dashboard:
-    link: "http://{host}:{port}/status"
+    link: "{scheme}://{host}:{port}/status"
```

Comment:
scheme -> protocol?

(scheme may also be a standard term here, I just haven't heard of it before)

Reply:
Scheme is what `urllib.parse.urlparse` uses.

Reply:
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Generic_syntax

Reply:
Alrighty then.  Thanks for the reference.